travis-ymls/tblib.travis.yml

```diff
@@ -0,0 +1,112 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : tblib
+# Source Repo         : https://github.com/ionelmc/python-tblib
+# Travis Job Link     : https://travis-ci.com/github/santosh653/pylibdmtx
+# Created travis.yml  : No
+# Maintainer          : Santosh Kulkarni <santoshkulkarni70@gmail.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+language: python
+dist: xenial
+cache: false
+env:
+  global: SEGFAULT_SIGNALS=all
+    
+matrix:
+  include:
+    - python: '3.6'
+      env:
+        - TOXENV=check
+      arch: amd64
+      global: LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
```

Comment:
global is not a valid keyword within matrix. Also the travis link is pointing to wrong path https://travis-ci.com/github/santosh653/pylibdmtx